README.md

```diff
@@ -52,6 +53,22 @@ npm link
   every time a file in the current working directory changes.
 * `jpm sign` Retrieve a Mozilla-signed `.xpi` file for your current add-on.
 
+### Fork features
```

Comment:
I change the heading.
P.S. best place for this section is  addon manifest documentation